articles/storage/blobs/versioning-enable.md

```diff
@@ -92,6 +92,9 @@ For more information about deploying resources with templates in the Azure porta
 
 ---
 
+> [!IMPORTANT]
```

Comment:
Thanks Amrinder! Seems like this is a limitation that is specific to the portal experience as the UI no longer shows the option to modify retention. That said, what about PowerShell or CLI? Can't folks still use command parameters to modify the retention setting? If so, does that configuration change automatically remove the Lifecycle Management policy. If this is true, then the note would recommend that they can either manually remove the policy or use PowerShell / CLI to modify the setting. If you don't know, I can test.

Reply:
Hi @normesta - I haven't tested that yet. I can try from my side tomorrow and share my findings.

Reply:
Hi @normesta - I tried and this option isn't available for PS so probably this behavior in Portal only. 

In the cmdlet also, there isn't any option to set that retention for version similar to what we have for change feed. So, I believe, that's from UI, it is not possible and also not from PS as well.

https://learn.microsoft.com/en-us/powershell/module/az.storage/update-azstorageblobserviceproperty?view=azps-10.1.0

One workaround is to edit/update the policy configuration but that could be a lengthy process.

Reply:
Gotcha. I also verified. Thanks. Seems like this note should appear in the portal section only. Right now you have it outside of the portal tab (in the doc). No worries. I don't want to keep hounding you. I'll sign off + go back and move the block myself. Cheers